astrobee/survey/survey_manager/src/survey_manager/command_astrobee.py

```diff
@@ -71,43 +71,43 @@ def first_non_zero(a: int, b: int) -> int:
 
 def exposure_change(config_static, bay_origin, bay_destination):
     # Going to JEM
-    if bay_origin == "nod2_hatch_to_jem" and bay_destination == "jem_hatch_from_nod2":
+    if bay_origin == "nod2_bay1" and bay_destination == "jem_bay0":
```

Comment:
we should add the map and exposure transitions to the config files instead of hardcoding it